### ISSUE TITLE
Getting started needs some clarification

### ISSUE THREAD

**Issue by mapfel** [fsprojects/Paket#3546]
### Description

[Getting started](https://fsprojects.github.io/Paket/getting-started.html) states in step 5:

> After the first .paket/paket.exe invocation Paket will create a couple of files in .paket — commit those as well.

Without any parameters that throw:

	Paket version 5.201.1
	no valid subcommand has been specified.

### Repro steps

- Follow steps as described in Getting started
- run the provided statement

### Expected behavior

- creation of some assets as described in the documentation

### Actual behavior

- output `no valid subcommand has been specified`
- nothing else

### Known workarounds

- no workaround known

**Comment by forki** [comment]
@matthid @agross can anyone remember what we wanted to say in that part of the docs?

**Comment by agross** [comment]
I think the docs are referring to the targets file.

**Comment by agross** [comment]
I too think the docs are unclear about those files, namely `paket.targets` and `Paket.Restore.targets`. I only commit the former and `.gitignore`d the latter.

**Comment by Freshdouble** [comment]
What is the first .paket/packet.exe invocation at that point?

**Comment by agross** [comment]
paket init or convert-from-nuget or auto-restore --on are the usual
suspects.

**Comment by mapfel** [comment]
Yes, one of these subcommands will work depending upon the current starting point. But for a _Getting Started_ tutorial it should provide a working subcommand.

During the first steps with Paket you don't have a clue what's going on and you like to get a first impression. Running into an error during your first steps creates frustrations, which often enough let you stop from going deeper into a topic. The first impression counts.
